transmart-rest-api/grails-app/controllers/org/transmartproject/interceptors/DimensionAuditInterceptor.groovy

```diff
@@ -0,0 +1,17 @@
+package org.transmartproject.interceptors
+
+import groovy.transform.CompileStatic
+
+@CompileStatic
+class DimensionAuditInterceptor extends AuditInterceptor {
```

Comment:
A separate audit interceptor class for every controller... isn't that a bit too much?

Reply:
hmm, ok, I wanted them to be more descriptive, but if it is not needed, I can move everything to one interceptor that will log information containing message with a controller name and action. Would it be sufficient?

Reply:
The only disadvantage is that an action name is not always describing the action well, but you mentioned they do not have to be readable in UI, so should be fine.